collections/guides/encrypt_us_bank_accounts/index.md

```diff
@@ -0,0 +1,218 @@
+app.post('/create', async (req, res) => {
+    const {accountNumber, routingNumber} = req.body;
+
+    const atomicBank = await basisTheory.atomicBanks.create({
+        bank: {
+            routingNumber,
+            accountNumber
+        }
+    });
+
+    account = atomicBank;
+
+    res.send()
+})
+```
+
+To test that you're now storing this information, you can run this curl command:
+```bash
+curl --location --request POST 'http://127.0.0.1:3000/create' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "routingNumber": "021000021",
+    "accountNumber": "1234567891099"
+}'
+```
+
+## Retrieve and return the raw data
+
+After we've stored the information with Basis Theory, we will only be storing a reference to the raw bank account information. With that in mind, we will alter our `/get` endpoint to decrypt and retrieve the bank information and return it from our API:
+
+```js
+app.get('/get', async (req, res) => {
+    const atomicBank = await basisTheory.atomicBanks.retrieveDecrypted(account.id);
+    res.send(atomicBank.bank)
+})
+```
+
+To test you're retrieving the raw data from your updated endpoint: 
+
+_(keep in mind: every time the sample app restarts you'll need to call `/create`)_ 
+```bash
+curl --location --request GET 'http://127.0.0.1:3000/get'
+
+/// { "routingNumber": "021000021", "accountNumber": "1234567891099" }
+```
+
+Success! You are now securely storing your data with Basis Theory *(who will manage the encryption and keep the data safe on your behalf)*!
+
+## Optional: Add an endpoint to return masked data
+
+Your system may now be safely storing bank information, but how do you make your systems even safer?
+
+Basis Theory by default returns `mask` bank information back to your system, so you're able to store and use this non-sensitive data. These masks allow your systems to ONLY use the raw decrypted bank account data when you absolutely need it!
+
+Below is a new endpoint, showing how you can return the masked data stored in your system:
+```js
+app.get('/get_mask', (req, res) => {
+    res.send(account.bank)
+})
+```
+
+To test you're retrieving the masked data from your new endpoint:
+
+_(keep in mind: every time the sample app restarts you'll need to call `/create`)_
+```bash
+curl --location --request GET 'http://127.0.0.1:3000/get_mask'
+
+/// { "routingNumber": "021000021", "accountNumber": "XXXXXXXXX1099" }
+```
+
+## Wrap up
+
+You're now able to quickly update your existing systems to encrypt and safely store bank data with Basis Theory, you may now be asking "what's next?" 
+
+- Use our Serverless Reactors to create an ACH file (if you're looking to do this, [reach out!](https://basistheory.com/contact))
+- [Never have your systems touch bank data again](/guides/use-us-bank-accounts-without-touching-them) 
+- Interact with any other banking service you can imagine (e.g. Dwolla, Plaid, etc)
+- [Easily encrypt your existing bank data](https://docs.basistheory.com/api-reference/#tokenize)

```

Comment:
this will go to the Bulk guide when we get it out! 

https://linear.app/basis-theory/issue/ENG-1903/bulk-token-guide-create-a-guide-for-bulk-tokenization